test/unittests/evm_test.cpp

```diff
@@ -95,6 +95,40 @@ TEST_F(evm, sub_and_swap)
     EXPECT_EQ(result.output_data[31], 1);
 }
 
+TEST_F(evm, swapsn_jumpdest)
+{
+    // Test demonstrating possible problem with introducing multibyte SWAP/DUP instructions as per
+    // EIP-663 variants B and C.
+    // When SWAPSN is implemented execution will fail with EVMC_BAD_JUMP_DESTINATION.
+    const auto swapsn = "b3";
+    const auto code = push(4) + OP_JUMP + swapsn + OP_JUMPDEST + push(0) + ret_top();
+
+    rev = EVMC_PETERSBURG;
+    execute(code);
+    EXPECT_STATUS(EVMC_SUCCESS);
+
+    rev = EVMC_ISTANBUL;
```

Comment:
Can you add one more execution with `EVMC_MAX_REVISION` to detect possible problems in future revisions?

Reply:
@chfast added